From 0f394d64af038693c9d1345c0efc79cfa9089099 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Erdmann?= <dywi@mailerd.de>
Date: Sat, 8 Aug 2015 23:49:19 +0200
Subject: [PATCH 1/1] fix: tlp-rf-func: add missing "&&"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: André Erdmann <dywi@mailerd.de>
---
 tlp-rf-func | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tlp-rf-func b/tlp-rf-func
index 5fdac12..22d39f7 100755
--- a/tlp-rf-func
+++ b/tlp-rf-func
@@ -178,7 +178,7 @@ device_switch () { # switch radio type state
     fi
 
     # wifi, wwan: quit if rfkill disabled
-    [ "$X_NMCLI_ONLY" = "1" ] && [ "$1" != "bluetooth" ] return 0
+    [ "$X_NMCLI_ONLY" = "1" ] && [ "$1" != "bluetooth" ] && return 0
 
     # if current state does not match desired state:
     # switch rfkill device state
-- 
2.4.5

